src/common/store/ui/share-modal/types.ts

```diff
@@ -0,0 +1,17 @@
+import { PayloadAction } from '@reduxjs/toolkit'
+
+import { ID } from 'common/models/Identifiers'
+import { Track } from 'common/models/Track'
+
+export type ShareModalState = {
+  isOpen: boolean
```

Comment:
I don't think you need `isOpen` in this slice since that is handle by the modal slice